### ISSUE TITLE
Would this be permissible by PTP staff?

### ISSUE THREAD

**Issue by ergusto** [winneon/ptpfreeleech#1]
I know they quite purposefully disallow filtering by freeleech in autodl-irssi, so it's possible they might look down on this. I couldn't find any mention of it in the forums, did you announce it anywhere? Was thinking of making something similar of my own but this looks pretty good. Have you been using it successfully on the tracker for all this time without issue?

**Comment by winneon** [comment]
I haven't asked the PTP staff directly about the use of my tool.

If you were to boil the tool down to 1 sentence, it just checks a regularly accessible URI that is found by navigating through existing menus (/torrents.php?action=advanced&freetorrent=1&grouping=0) and compares the data against user-customizable values. There's obviously more to the tool but that's basically what it does. You could bookmark that URI and you would be doing what the tool does.

I don't know if the PTP staff would frown upon use of the tool. If they indeed dislike use of autodl-irssi, it is possible that they would dislike this tool as well. I'll ask them and get back to you.

For now, I've been using this tool every day ever since it was first in development, which would be 4 months. I haven't run into any problems in that time.

**Comment by ergusto** [comment]
Ah awesome, thanks for the response. I was under the impression they stopped announcing freeleech in their IRC announce channel in order to 'level the playing field' or something along those lines, which is why I thought they might not be particularly happy with the farming freeleech aspect, not that they were against the use of autodl-irrsi generally. But that's good to hear, thanks. Presumably you just run this in cron?

**Comment by winneon** [comment]
Yes, I run it every 30 minutes in cron. That seems to be a good time interval to not stress their systems and also process every freeleech torrent.
